database/snowflake/requirements.txt

```diff
@@ -0,0 +1,2 @@
+snowflake-connector-python[pandas]
```

Comment:
Haven't seen this syntax. What does it mean?

Reply:
It's a notation used to install the pandas compatible version of snowflake. Ref: https://docs.snowflake.com/en/user-guide/python-connector-pandas.html#installation